docs/install.md

```diff
@@ -271,19 +273,33 @@ From the top-level netCDF-4 source directory, the following builds netCDF-4 with
     $ make install
 ~~~~
 
-If parallel I/O access to netCDF classic and 64-bit offset files is
-needed, an alternate
-[parallel-netcdf library](https://trac.mcs.anl.gov/projects/parallel-netcdf/wiki/WikiStart),
-referred to as "PnetCDF", must also be installed. Assume it was
-installed in the directory named by the PNDIR shell variable.
-Then, from the top-level netCDF-4 source directory, configure netCDF
-with the "--enable-pnetcdf" option:
+### Building PnetCDF from source {#build_pnetcdf_from_source}
+
+To enable parallel I/O support for classic netCDF files, i.e. CDF-1, 2 and 5
+formats, [PnetCDF library](https://parallel-netcdf.github.io) must also be
+installed.  First specify where you want to install PnetCDF in a shell
+variable, for example PNDIR, and build it from the PnetCDF top-level source
+directory. If you would like to build the shared library, include
+`--enable-shared` option at the configure command line.  By default, only a
+static library is built.
+
+~~~~{.py}
+    $ # Build and install PnetCDF
+    $ PNDIR=/usr/local
+    $ ./configure --prefix=${PNDIR} --with-mpi=/path/to/MPI/compilers
+    $ make check
+    $ make install   # or sudo make install, if root permissions required
+~~~~
+
+To build netCDF-4 with PnetCDF support, from the top-level netCDF-4 source
+directory, configure netCDF with the "--enable-pnetcdf" option. If PnetCDF
+is built with static library only, add "--disable-shared" option.
```

Comment:
Why does pnetcdf build with static libraries by default? Aren't shared libraries more performant on HPC systems?

Reply:
Quoted from a web page at NERSC. "Using dynamic libraries may introduce delays in application launch times. This delay is caused by the runtime linking process and the relative inefficiency of symbol lookup in distributed shared objects. Also,  there may be a small performance degradation during execution."
http://www.nersc.gov/users/computational-systems/edison/running-jobs/shared-and-dynamic-libraries/

Reply:
Interesting. Thanks. I will see if I can test this.